<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Decay Cascades</string>
  </property>
  <property name="windowIcon">
   <iconset resource="kaihen.qrc">
    <normaloff>:/edit-find.png</normaloff>:/edit-find.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{
    font-weight: bold;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="daughterGroupBox">
     <property name="title">
      <string>Daughter Mass Range</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="minALabel">
        <property name="text">
         <string>Minimal A:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>minA</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="minA">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>299</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="maxA">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>299</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="maxALabel">
        <property name="text">
         <string>Maximal A:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>maxA</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parent Constraints</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="parentHalfLifeLabel">
        <property name="text">
         <string>Half Life:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="minParentHalfLifeCheck">
          <property name="text">
           <string>Min.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HalfLifeSpinBox" name="minParentHalfLife">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="value">
           <double>86400.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QCheckBox" name="maxParentHalfLifeCheck">
          <property name="text">
           <string>Max.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HalfLifeSpinBox" name="maxParentHalfLife">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="value">
           <double>2592000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gammaGroupBox">
     <property name="title">
      <string>Gamma Constraints</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="minIntensCheck">
        <property name="text">
         <string>Minimal Gamma Intensity:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="minIntens">
        <property name="suffix">
         <string notr="true"> %</string>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="intermediateGroupBox">
     <property name="title">
      <string>Intermediate Level Constraints</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="intHalfLifeLabel">
        <property name="text">
         <string>Half Life:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="minIntHalfLifeCheck">
          <property name="text">
           <string>Min.</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HalfLifeSpinBox" name="minIntHalfLife">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000001000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="maxIntHalfLifeCheck">
          <property name="text">
           <string>Max.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HalfLifeSpinBox" name="maxIntHalfLife">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
          <property name="value">
           <double>0.000010000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="muValCheck">
        <property name="text">
         <string>Minimal Âµ Value:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="muVal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="qValCheck">
        <property name="text">
         <string>Minimal Q Value:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="qVal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="anisotropyGroupBox">
     <property name="title">
      <string>Minimal Anisotropies</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="a22Check">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="a22Label">
          <property name="text">
           <string>A&lt;sub&gt;22&lt;/sub&gt;</string>
          </property>
          <property name="buddy">
           <cstring>a22</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="a22">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="a24Check">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="a24Label">
          <property name="text">
           <string>A&lt;sub&gt;24&lt;/sub&gt;</string>
          </property>
          <property name="buddy">
           <cstring>a24</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="a24">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="a42Check">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="a42Label">
          <property name="text">
           <string>A&lt;sub&gt;42&lt;/sub&gt;</string>
          </property>
          <property name="buddy">
           <cstring>a42</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="a42">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="a44Check">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="a44Label">
          <property name="text">
           <string>A&lt;sub&gt;44&lt;/sub&gt;</string>
          </property>
          <property name="buddy">
           <cstring>a44</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="a44">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HalfLifeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>HalfLifeSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>minA</tabstop>
  <tabstop>maxA</tabstop>
  <tabstop>minParentHalfLifeCheck</tabstop>
  <tabstop>minParentHalfLife</tabstop>
  <tabstop>maxParentHalfLifeCheck</tabstop>
  <tabstop>maxParentHalfLife</tabstop>
  <tabstop>minIntensCheck</tabstop>
  <tabstop>minIntens</tabstop>
  <tabstop>minIntHalfLifeCheck</tabstop>
  <tabstop>minIntHalfLife</tabstop>
  <tabstop>maxIntHalfLifeCheck</tabstop>
  <tabstop>maxIntHalfLife</tabstop>
  <tabstop>muValCheck</tabstop>
  <tabstop>muVal</tabstop>
  <tabstop>qValCheck</tabstop>
  <tabstop>qVal</tabstop>
  <tabstop>a22Check</tabstop>
  <tabstop>a22</tabstop>
  <tabstop>a24Check</tabstop>
  <tabstop>a24</tabstop>
  <tabstop>a42Check</tabstop>
  <tabstop>a42</tabstop>
  <tabstop>a44Check</tabstop>
  <tabstop>a44</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="kaihen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minParentHalfLifeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>minParentHalfLife</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxParentHalfLifeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxParentHalfLife</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minIntensCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>minIntens</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minIntHalfLifeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>minIntHalfLife</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxIntHalfLifeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxIntHalfLife</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>muValCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>muVal</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qValCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>qVal</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a22Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>a22</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a24Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>a24</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a42Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>a42</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a44Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>a44</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
